core/src/main/java/org/renjin/eval/Context.java

```diff
@@ -144,7 +144,7 @@ public Context beginFunction(Environment rho, FunctionCall call, Closure closure
     context.parent = this;
     context.evaluationDepth = evaluationDepth+1;
     context.closure = closure;
-    context.environment = Environment.createChildEnvironment(closure.getEnclosingEnvironment());
+    context.environment = Environment.createChildEnvironment(closure.getEnclosingEnvironment()).build();
```

Comment:
In one place it came handy, in one other place usage required further implementation of findVariables, lock related and (active) binding related methods.